.github/workflows/pr-checks-push-images.yaml

```diff
@@ -0,0 +1,50 @@
+name: "PR checks - Push operator images"
+
+on:
+  pull_request_target:
+    branches:
+      - master
+
+env:
+  REGISTRY_PREFIX: quay.io/redhat-developer
+  REPO: servicebinding-operator
+
+jobs:
+  push-operator-images:
+    name: "Push operator images (PR)"
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout Git Repository
+        uses: actions/checkout@v2
+
+      - name: Setup CLI
+        uses: ./.github/actions/setup-cli
```

Comment:
Do we really need these two steps?

Reply:
We need the first for the `Setup local registry` step were we refering `.github/actions/setup-podman/setup-local-registry.sh` script and I've used the other to set up PATH, but that can be done in a standalone simple step.